subprojects/gradle/module-dependencies-graph/src/main/kotlin/com/avito/module/dependencies/ModuleDependenciesGraphPlugin.kt

```diff
@@ -11,8 +12,24 @@ public class ModuleDependenciesGraphPlugin : Plugin<Project> {
         require(target.rootProject == target) {
             "must be applied to the root project"
         }
-        target.tasks.register("findAndroidApp", FindAndroidAppTask::class.java)
+        target.tasks.register("findAndroidApp", FindAndroidAppTask::class.java) {
+            it.loggerFactory.set(
+                GradleLoggerFactory.fromProject(
+                    project = target,
+                    pluginName = "ModuleDependenciesGraphPlugin",
+                    taskName = "FindAndroidAppTask"
+                )
+            )
+        }
         target.tasks.register("collectAppsMetrics", CollectAppsMetricsTask::class.java) {
+            it.loggerFactory.set(
+                GradleLoggerFactory.fromProject(
+                    project = target,
+                    pluginName = "ModuleDependenciesGraphPlugin",
+                    taskName = "CollectAppsMetricsTask"
+                )
```

Comment:
I propose to avoid hardcode by hiding this frequent use case:

```kotlin
GradleLoggerFactory.fromProject(project, plugin, task) // <-- no strings here
```

Reply:
Yep, fixed